tests/components/deconz/test_device_trigger.py

```diff
@@ -135,14 +130,299 @@ async def test_get_triggers(hass, aioclient_mock):
     assert_lists_same(triggers, expected_triggers)
 
 
+async def test_get_triggers_manage_unsupported_remotes(hass, aioclient_mock):
+    """Verify no triggers for an unsupported remote."""
+    data = {
+        "sensors": {
+            "1": {
+                "config": {
+                    "alert": "none",
+                    "group": "10",
+                    "on": True,
+                    "reachable": True,
+                },
+                "ep": 1,
+                "etag": "1b355c0b6d2af28febd7ca9165881952",
+                "manufacturername": "IKEA of Sweden",
+                "mode": 1,
+                "modelid": "Unsupported model",
+                "name": "TRÅDFRI on/off switch ",
+                "state": {"buttonevent": 2002, "lastupdated": "2019-09-07T07:39:39"},
+                "swversion": "1.4.018",
+                "type": "ZHASwitch",
+                "uniqueid": "d0:cf:5e:ff:fe:71:a4:3a-01-1000",
+            }
+        }
+    }
+    with patch.dict(DECONZ_WEB_REQUEST, data):
+        await setup_deconz_integration(hass, aioclient_mock)
+
+    device_registry = await hass.helpers.device_registry.async_get_registry()
+    device = device_registry.async_get_device(
+        identifiers={(DECONZ_DOMAIN, "d0:cf:5e:ff:fe:71:a4:3a")}
+    )
+
+    assert device_trigger._get_deconz_event_from_device_id(hass, device.id)
```

Comment:
Can we test this with an automation or public API of automation or device_automation integrations?

Reply:
This is just to verify that the helper behaves as expected

Reply:
Removed them